packages/dashboard/src/features/dashboard/components/connect/MCPSection.tsx

```
@@ -132,13 +184,36 @@ export function MCPSection({
               className={cn('bg-semantic-0', isLoading && 'animate-pulse')}
             />
           )}
+
+          {/* CLI mode content */}
+          {showCliCommand && (
+            <div className="flex flex-col gap-2">
+              <CodeBlock
+                code={cliConnectCommand}
+                label="Connect"
+                className="bg-semantic-0"
+              />
```

ESLint

[error] Replace `⏎················code={cliConnectCommand}⏎················label="Connect"⏎················className="bg-semantic-0"⏎·············` with `·code={cliConnectCommand}·label="Connect"·className="bg-semantic-0"`

(prettier/prettier)